Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -1456,7 +1615,7 @@
           {
             "type": 1,
             "content": {
-              "json": "📄 Pageview ​ ‍ 📅 Event ​ ‍ 💬 Request"
+              "json": "������ Pageview ​ ‍ ������ Event ​ ‍ ������ Request"
```

Comment:
Did you mean to change the emoji? (Here and in other places below)

Reply:
Thx. It looks like something wring. I will update it